src/svg/test.svg

```diff
@@ -0,0 +1,39 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1849" height="949">
```

Comment:
is this a local test file, or one that you mean to check in? if it's the latter, maybe move to a top-level `testfiles` directory